keras_benchmark/scripts/models/resnet50_benchmark.py

```diff
@@ -0,0 +1,91 @@
+"""Benchmark a Resnet50 model
+Credit:
+Script modified from TensorFlow Benchmark repo:
+https://github.com/tensorflow/benchmarks/blob/keras-benchmarks/scripts/keras_benchmarks/models/resnet50_benchmark.py
+"""
+
+from __future__ import print_function
+
+import time
+
+import numpy as np
+from models import timehistory
+
+import keras
+
+
+def crossentropy_from_logits(y_true, y_pred):
```

Comment:
why is this a global method? make it private if its not being used outside of the class